parking-lot-help-request/README.md

```diff
@@ -0,0 +1,118 @@
+# Parking Lot Help Request
+
+Host-free low latency notification system to alert attendants in a parking lot or facility that customers need assistance. To run this project youself you'll need to:
+
+* [Purchase the necessary hardware and configure it](#hardware).
+* [Configure Notehub](#notehub).
+* [Configure the project's Notecard](#notecard-firmware).
+* [Set up a Twilio account and configure the data to route from Notehub to Twilio](#twilio-firmware).
+## Hardware
+
+The following hardware is required to run the Parking Lot Help Request application.
+
+* [Notecarrier A with pre-soldered headers](https://shop.blues.io/products/carr-al)
+* [Notecard LTE Cat-1](https://shop.blues.io/products/note-wbna-500) (for use in North America)
+* [Sparkfun RGB LED breakout - WS2812B](https://www.sparkfun.com/products/13282)
+* [Twidec 12mm momentary push button with pre-soldered wires](https://www.amazon.com/gp/product/B08JHW8BPV/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&th=1)
+* [LiPo battery](https://www.adafruit.com/product/2011)
+* [Solar panel with JST connectors (any of these are compatible)](https://www.seeedstudio.com/catalogsearch/result/?q=solar%20panels)
```

Comment:
Might be worth saying which specific one you used. “Any of these work. We went with the ...”

Reply:
Done.